### DISCUSSION TITLE
commitlerim gözükmüyor

### DISCUSSION THREAD

**Discussion by batuhanbrbrgl** [community/community#14523] [Other Feature Feedback, Questions, & Ideas]
VSC Aracılığıyla yaptığım çalışmalar profilim de gözükmüyor acaba sebebi nedir ?
![sssss](https://user-images.githubusercontent.com/76756646/162348570-0a4717d7-7aec-4219-bddb-638b3cc237c5.JPG)

**Comment by tolgaacgul** [comment]
Merhaba çözümünü bulabildiniz mi? Aynı sorunu bende yaşıyorum.

**Reply by P1kKaP0u** [reply]
Merhaba ben de aynı sorunu yaşıyordum şu şekilde çözdüm. Git  bash de commit ettiğim email adresi ile github hesabının mail adresi aynı olması gerekiyor

**Reply by tolgaacgul** [reply]
Evet sorun şu şekil çözülebilir o zaman.
`git config user.email` kodu ile yereldeki repoda epostanızı kontrol edebilirsiniz.
`git config user.email kullaniziadi@gmail.com` kodu ile eposta değişikliği yapılabilir.

**Reply by OnurKor** [reply]
Teşekkür ederim sayenizde bende çözdüm sorunumu.

**Reply by tolgaacgul** [reply]
Rica ederim. İyi kodlamalar

**Reply by FatihOzkan0** [reply]
Visiual studio üzerinden commitleme yapıyorsak nasıl çözücez sorunu

**Reply by ademk7604** [reply]
cok tesekkur ederi hocam

**Reply by Kyawgyi99999** [reply]
Really appreciate your help me

**Reply by yusufakbs** [reply]
Mail adresinizi yazdığınız halde hala görünmüyor ise mail ayarlarından "`Keep my email addresses private`"kutucuğunu işaretlemişsiniz demektir. Bu alanın altında sizin için tanımlanmış git işlemlerini gerçekleştirebilmeniz adına "`11111111+test@users.noreply.github.com`" benzeri bir mail adresi olacaktır. Bunun ile `config` işlemi gerçekleştirirseniz sorununuzu çözecektir.

**Reply by elifery06** [reply]
git bash de ;
 git config --global user.email 
 ifadesi boş çıktı veriyorsa mail atanmamış demektir.Mail atamak için
 git config --global user.email "mail adresiniz"
 yazarsanız hata düzeldi bende .
